### DISCUSSION TITLE
Questions about versioning feature

### DISCUSSION THREAD

**Discussion by tpluscode** [inception-project/inception#4649] [Q&A]
I found the flag to enable git versioning.

Can it be used to backup/restore a project to any past state?
Should Git Large File Storage be enabled in the target repository?

**Comment by reckart** [comment]
It is a quite experimental feature and can currently only be used to create a snapshot of the annotations and the layer configuration. It is not possible to restore the data from the UI. I would say, in the current state, it is not terribly useful, but could be used as a basis for further improvements.
